ui/component/notification/index.js

```diff
@@ -1,8 +1,10 @@
 import { connect } from 'react-redux';
```

Comment:
When a "comment notification" is clicked, it navigates to the claim page but also reloads the site, causing the cache to be cleared and re-authenticated. Should avoid a reload if possible.